satellite/peer.go

```diff
@@ -352,6 +357,11 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 		pb.RegisterKadInspectorServer(peer.Server.PrivateGRPC(), peer.Kademlia.Inspector)
 	}
 
+	{ // setup contact service
+		log.Debug("Setting up contact service")
+		peer.Contact.Service = contact.NewService(peer.Log.Named("contact"), peer.Overlay.Service, peer.Transport)
```

Comment:
do we need an endpoint for the SA peer as well that has the contact.PingNode() rpc?

Reply:
I don't think so. I'm worried if we did that, that it would be confusing for PingNode to be serviced by contact.Service on the nodes and contact.Service on the satellites, but where those two contact.Service's are totally different things :/

Reply:
lol. im confused. ok well lets merge this and sort it out in my next ticket.

Reply:
Heh. I mean, cause, storagenode/contact.Service (or Endpoint) and satellite/contact.Service (or Endpoint) are totally different things